client/src/app/components/ConfirmDeleteCatalog/ConfirmDeleteCatalog.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { FC, useState } from "react";
+import { useTranslation, Trans } from "react-i18next";
+import {
+  ModalProps,
+  Button,
+  ButtonVariant,
+  Modal,
+  ModalVariant,
+  Text,
+  TextInput,
+} from "@patternfly/react-core";
+
+import "./ConfirmDeleteCatalog.css";
+
+type ConfirmDeleteDialogProps = {
+  cancelBtnLabel?: string;
+  deleteBtnLabel?: string;
+  deleteObjectMessage: string;
+  isOpen: boolean;
+  nameToDelete?: string;
+  onClose: () => void;
+  onConfirmDelete: () => void;
+  titleWhat: string;
+  titleIconVariant?: ModalProps["titleIconVariant"];
+};
+
+const ConfirmDeleteDialog: FC<ConfirmDeleteDialogProps> = ({
+  cancelBtnLabel,
+  deleteBtnLabel,
+  deleteObjectMessage,
+  isOpen,
+  nameToDelete,
+  onClose,
+  onConfirmDelete,
+  titleWhat,
+  titleIconVariant = "warning",
+}) => {
+  const { t } = useTranslation();
+
+  const [nameToDeleteInput, setNameToDeleteInput] = useState<string>("");
+
+  const isDisabled = nameToDeleteInput !== nameToDelete;
+
+  const handleClose = () => {
+    setNameToDeleteInput("");
+    onClose();
+  };
+
+  const confirmBtn = (
+    <Button
+      id="confirm-delete-dialog-button"
+      key="confirm"
+      aria-label="confirm"
+      variant={ButtonVariant.danger}
+      isDisabled={isDisabled}
+      onClick={isDisabled ? undefined : onConfirmDelete}
+    >
+      {deleteBtnLabel ?? t("actions.delete")}
+    </Button>
+  );
+
+  const cancelBtn = (
+    <Button
+      key="cancel"
+      id="cancel-delete-button"
+      aria-label="cancel"
+      variant={ButtonVariant.link}
+      onClick={handleClose}
+    >
+      {cancelBtnLabel ?? t("actions.cancel")}
+    </Button>
+  );
+  return (
+    <Modal
+      id="confirm-delete-dialog"
+      variant={ModalVariant.small}
+      titleIconVariant={titleIconVariant}
+      isOpen={isOpen}
+      onClose={handleClose}
+      aria-label={t("Confirm delete dialog")}
```

Comment:
Either this key needs to be fixed to something like `"dialog.aria.confirmDelete" or just use the English string.  I've not seen any other externalized aria labels.

Reply:
Right, habit from CNV, will change